Civilla-Project/Police-Control/src/main/java/BotActions.java

```diff
@@ -0,0 +1,186 @@
+import java.util.ArrayList;
+
+interface BotAction {
+    String execute(BotSession session, String input);
+}
+
+class AdminProfileStatus implements BotAction {
+    @Override
+    public String execute(BotSession session, String input) {
+        switch (input) {
+            case "Policeman":
+                session.isPoliceman = true;
+                session.baseCommand = CommandTrees.policeTree;
+                session.currentCommand = session.baseCommand;
+                return "Now you are a policeman.";
+            case "Citizen":
+                session.isPoliceman = false;
+                session.baseCommand = CommandTrees.basicTree;
+                session.currentCommand = session.baseCommand;
+                return "Now you are a citizen.";
+            default:
+                return "Unsupported input";
+        }
+
+    }
+}
+
+class GoToLabel implements BotAction {
+    @Override
+    public String execute(BotSession session, String input) {
+        BotCommand newBotCommand = session.currentCommand.getCommandByLabel(input);
+        if (newBotCommand != null) {
+            session.setCurrentCommand(newBotCommand);
+        } else if (!input.equals("back"))
+            return "Unsupported input";
+        return "";
+    }
+}
+
+
+class PutFnameToDatabase implements BotAction {
+    @Override
+    public String execute(BotSession session, String input) {
+        CivillaDatabaseItem item;
+        if (!session.tempItems.isEmpty()) {
+            item = session.tempItems.get(0);
+        } else {
+            item = new CivillaDatabaseItem(null, null, null, 0, false);
```

Comment:
Maybe better to add default constructor with 0 arguments?

Reply:
Nice one, I'll do it.